mycroft/configuration/config.py

```diff
@@ -152,3 +157,75 @@ def get_config():
         if not ConfigurationManager._config:
             ConfigurationManager.load()
         return ConfigurationManager._config
+
+
+class ConfigBuilder(object):
+    """
+    ConfigBuilder, used to construct list of configuration files.
+    """
+    def __init__(self):
+        """
+            Creates an empty ConfigBuilder object.
+        """
+        self.default = []
+        self.system = []
+        self.user = []
+        self.custom = []
+        self.ptr = self.custom
```

Comment:
It's for the .append() method. Continue adding at the last point, this is strictly not needed.